Source/Extensions/Blazorise.DataGrid/BaseDataGridColumn.cs

```diff
@@ -18,6 +18,11 @@ public class BaseDataGridColumn<TItem> : BaseDataGridComponent
     /// <returns>Formatted display value.</returns>
     public string FormatDisplayValue( object value )
     {
+        if (value is string[] values )
```

Comment:
**Reminder to self :** Can we perhaps try to just find whether it's a collection and enumerate over it while doing .ToString to support more types?